src/randoop/sequence/AbstractGenerator.java

```diff
@@ -113,41 +117,50 @@
   public static Sequence currSeq = null;
   
   /**
-   * The list of final sequences that are printed out as JUnit tests (i.e. Randoop's output). 
+   * The list of failure sequences to be saved as JUnit tests
+   */
+  public List<ExecutableSequence> outFailureSeqs = new ArrayList<>();
```

Comment:
The command-line arguments call these error tests.  Please use consistent terminology throughout the documentation and codebase.